package-lock.json

```diff
@@ -1,9901 +1,8 @@
 {
   "name": "bedrock",
   "version": "0.1.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
@stevejalim looks like you might be running an outdated version of NPM here as the lockfile has reverted to an older format?

Reply:
🤦 thanks -- fixed!